spark-operator-api/src/main/java/org/apache/spark/k8s/operator/status/ApplicationStateSummary.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.spark.k8s.operator.status;
+
+import java.util.Set;
+
+public enum ApplicationStateSummary implements BaseStateSummary {
+  /** Spark application is submitted to the cluster but yet scheduled */
+  SUBMITTED,
```

Comment:
Is this the first place to use all capital in this repository?

For example, `RestartPolicy` and `ResourceRetainPolicy` of this PR has `CamelCase` instead of all caps.

It seems that we had better follow K8s standard consistently for `enum` by switching `ApplicationStateSummary` to `CamelCase`. WDYT. @jiangzho ?